core/templates/dev/head/domain/exploration/SolutionObjectFactory.ts

```diff
@@ -108,6 +108,10 @@ export class Solution {
       '". ' + explanation + '.');
   }
 
+  setAnswerExclusivity(answerIsExclusive: boolean) {
```

Comment:
This is not used anywhere in the codebase right? I don't think we need this function...

Reply:
I created this function only for tests purpose, but actually is not a good thing to do. I created it to check if the `getSummary` function would evalutes the `this.answerIsExclusive` to return `The only` or `One` string.

Reply:
This is a good practice in general. Though, if you do this, then all places which directly set/get the answerIsExclusive variable should use this function.

Reply:
I checked out and the set is not being used anywhere, so I'll refactored the `it` block where I use `setAnswerExclusivity` function.

Reply:
Done.